integration_tests/test_pointmatch_integrated.py

```diff
@@ -181,22 +176,23 @@ def test_get_match_groupIds_to_only(render, test_pm_collection):
 
 def test_delete_point_matches_between_groups(render):
     collection = 'test_delete'
-    owner = 'test'
+    # owner = 'test'
     renderapi.pointmatch.import_matches(
         collection, test_matches, render=render)
-    group1 = '0'
-    group2 = '1'
+    # group1 = '0'
+    # group2 = '1'
     renderapi.pointmatch.delete_point_matches_between_groups(
         collection, '0', '1', render=render)
     groups = renderapi.pointmatch.get_match_groupIds(collection, render=render)
     assert len(groups) == 2
 
+
 def test_delete_collection(render):
     collection = 'test_delete_collection'
-    owner = 'test'
+    # owner = 'test'
```

Comment:
why aren't we just nuking these lines?

Reply:
Yeah, I wanted to ask about those... if you don't think they're useful they can go.